src/client/util.js

```diff
@@ -16,19 +17,21 @@ export function isAndroidApp () {
   return typeof AndroidBridge !== 'undefined'
 }
 
-/* usage:
- * var connectWebViewJavascriptBridge = require('webViewJavascriptBridge')
- * connectWebViewJavascriptBridge(function(bridge) {
- *   bridge.send(...)
- * }
- */
-export function connectWebViewBridge (callback) {
-  if (window.WebViewJavascriptBridge) {
-    callback(window.WebViewJavascriptBridge)
-  } else {
-    document.addEventListener('WebViewJavascriptBridgeReady', function () {
-      callback(window.WebViewJavascriptBridge)
-    }, false)
+export function calliOSBridge (message, callback) {
+  if (isiOSApp()) {
+    var randomName = () => {
+      return randomstring.generate({length: 10, charset: 'alphabetic'})
+    }
+
+    if (callback) {
+      const callbackIdentifier = randomName()
+      if (!window.iOSCallbacks) {
+        window.iOSCallbacks = {}
+      }
+      window.iOSCallbacks[callbackIdentifier] = callback
```

Comment:
adding this to the global namespace is a little funky, but it was the only way I could figure out to make the callback accessible in wkwebview (and it seems to be the way wkwebview handles JS in general, see below `window.webkit...`)